org.eclipse.xtext.ui.shared/src/org/eclipse/xtext/ui/shared/Access.java

```diff
@@ -44,6 +44,9 @@ public T get() {
 			if (Activator.getDefault()==null) {
 				throw new IllegalStateException("The bundle has not been started!");
 			}
+			if (Activator.getDefault().getInjector()==null) {
```

Comment:
A matter of taste, but here I'd prefer local variables instead of subsequent calls to getDefault and getInjector